small spelling and grammar fixups to news285 (#1475)
bitcoinops/bitcoinops.github.io: _posts/en/newsletters/2024-01-17-newsletter.md

  any onchain fees directly <!-- endogenously --> included in the
  commitment transaction and the presigned HTLC-Success and HTLC-Timeout

>   transactions (HTLC-X transactions).  If the funding party doesn't have
>   many funds in the channel and if feerates rise, the funding party may

Reviewer: "Much" seems right to me. Isn’t _funds_ an uncountable noun? Either way, perhaps:

```suggestion
  transactions (HTLC-X transactions).  If the funding party is low on
  funds in the channel and if feerates rise, the funding party may
```
